samples/msal-node-samples/ElectronSystemBrowserTestApp/redirect/.npmrc

```diff
@@ -1 +0,0 @@
-package-lock=false
```

Comment:
Please restore at the root

Reply:
Already exist in the [root ](https://github.com/salman90/microsoft-authentication-library-for-js/blob/update-electron-system-browser/samples/msal-node-samples/ElectronSystemBrowserTestApp/.npmrc)